parties/githubclient.go

```diff
@@ -149,77 +147,69 @@ func (g *GithubOp) RequestTokens(cicHash string) ([]byte, error) {
 	return []byte(jwt.Value), err
 }
 
-func (g *GithubOp) VerifyPKToken(pkt *pktoken.PKToken, cosPk crypto.PublicKey) (map[string]any, error) {
-	if !pkt.OpSigGQ {
-		return nil, fmt.Errorf("non-GQ signatures not supported for github")
+func (g *GithubOp) VerifyPKToken(pkt *pktoken.PKToken, cosPk crypto.PublicKey) error {
+	sigType, ok := pkt.ProviderSignatureType()
+	if !ok {
+		return fmt.Errorf("provider signature type missing")
+	}
+
+	if sigType != pktoken.Gq {
+		return fmt.Errorf("github only support gq signatures")
 	}
 
-	cicphJSON, err := util.Base64DecodeForJWT(pkt.CicPH)
+	cic, err := pkt.GetCicValues()
 	if err != nil {
-		return nil, err
+		return err
 	}
 
-	expectedAud := string(util.B64SHA3_256(cicphJSON))
+	commitment, err := cic.Commitment()
```

Comment:
what about `cicHash, err := cic.Hash()`?

Reply:
I'm good with `Hash()`, but then I would make it so that we're actually injecting meaning in the value:
```golang
	commitment, err := cic.Hash()
```

Reply:
99d9763